From cf07365002d83c5a5b9cd266a45acd0023c84610 Mon Sep 17 00:00:00 2001
From: James Slagle <jslagle@redhat.com>
Date: Fri, 31 Jan 2014 07:28:20 -0500
Subject: [PATCH 01/36] Install openstack-swift-object

We need to install the swift object service package for our setup.

Change-Id: I95356bb9ffcb267c6fcddf471c7cafeaf4c05aad
---
 .../swift-storage/install.d/swift-package-install/76-swift-storage     | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/elements/swift-storage/install.d/swift-package-install/76-swift-storage b/elements/swift-storage/install.d/swift-package-install/76-swift-storage
index 1a6e21b..7ba7a8c 100755
--- a/elements/swift-storage/install.d/swift-package-install/76-swift-storage
+++ b/elements/swift-storage/install.d/swift-package-install/76-swift-storage
@@ -5,4 +5,5 @@ set -eux
 install-packages \
     openstack-swift-storage \
     openstack-swift-account \
-    openstack-swift-container
+    openstack-swift-container \
+    openstack-swift-object
-- 
1.8.5.3

